README.md

```diff
@@ -1 +1,33 @@
-# Including Submodules
\ No newline at end of file
+# Including Submodules
+
+## Objectives
+
+- Define a new Angular app module
+- Invoke Angular with angular.bootstrap
+- Invoke Angular with ng-app
+- Import the ngRoute submodule
+
+## Instructions
+
+Setup your directory structure as follows:
+
+- js/
+  - app/
+    - controllers/
+      - MainController.js
+    - app.js
+  - angular.js
+  - angular-route.js
+- index.html
+
+You can find `index.html`, `angular.js`, `angular-route.js` in this repo.
```

Comment:
Include those files in a `js` directory to begin with - then they can create the `app` directory inside of the `js` directory